superset-frontend/src/SqlLab/actions/sqlLab.js

```diff
@@ -1300,3 +1302,23 @@ export function createCtasDatasource(vizOptions) {
       });
   };
 }
+
+export function queryEditorSetFunctionNames(queryEditor, dbId) {
+  return function (dispatch) {
+    return SupersetClient.get({
+      endpoint: encodeURI(`/api/v1/database/${dbId}/function_names/`),
+    })
+      .then(({ json }) =>
+        dispatch({
+          type: QUERY_EDITOR_SET_FUNCTION_NAMES,
+          queryEditor,
+          functionNames: json.function_names,
+        }),
+      )
+      .catch(() =>
+        dispatch(
+          addDangerToast(t('An error occurred while fetching function names')),
```

Comment:
nit: there should be a period in this sentence

Reply:
Will fix, thanks!